05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,705 @@
+
+To specify this design, use the following syntax:
+
+```{r}
+#| label: recs
+#| eval: TRUE
+#| cache: TRUE
+recs_in <- read_csv(here::here("RawData", "RECS_2015", "recs2015_public_v4.csv"))
+
+des_recs <- recs_in %>%
+  as_survey_rep(weights=NWEIGHT, repweights=BRRWT1:BRRWT96, type="Fay",
+                rho=0.5, mse=TRUE, variables=c(DOEID, TOTALDOL, TOTSQFT_EN, REGIONC))
+
+des_recs
+
+summary(des_recs)
+```
+
+In specifying the design, the `variables` option was also used to include which variables might be used in analyses. This is optional but can make your object smaller. When printing the design object or looking at the summary, the replicate weight type is re-iterated as `Fay's variance method (rho= 0.5 ) with 96 replicates and MSE variances.` and the variables are included. No weight or probability summary is included as was done in some other design objects.
+
+### Jackknife method
+
+There are two jackknife estimators implemented in {srvyr} - Jackknife 1 (JK1), Jackknife n (JKn), and Jackknife 2 (JK2). The JK1 method can be used for unstratified designs and replicates are created by removing one PSU at a time so the number of replicates is the same as the number of PSUs If there is no clustering, then the PSU is the ultimate sampling unit (e.g., unit). 
+
+The JKn method is used for stratified designs and requires 2 or more PSUs per stratum. In this case, each replicate is created by deleting one PSU from each stratum so the number of replicates is the number of total PSUs across all strata. The JK2 method is a special case of JKn when there are exactly 2 PSUs sampled per stratum. For variance estimation, scaling constants must also be specified.
+
+#### The math {-}
+
+For the JK1 method, the standard error estimate for $\hat{\theta}$ is calculated as:
+
+$$se(\hat{\theta})=\sqrt{\frac{R-1}{R} \sum_{r=1}^R \left( \hat{\theta}_r-\hat{\theta}\right)^2}$$
+The JKn method is a bit more complex but the coefficients are generally provided. Within the stratum that had a PSU removed, the weights are adjusted by $n_h/(n_h-1)$ where $n_h$ is the number of PSUs in the stratum. The coefficients in other strata are 1. Let the coefficient that results for replicate $r$ be $\alpha_r$ then the standard error estimate for $\hat{\theta}$ is calculated as:
+
+$$se(\hat{\theta})=\sqrt{\sum_{r=1}^R \left(\alpha_r \hat{\theta}_r-\hat{\theta}\right)^2}$$
+
+#### The syntax {-}
+
+To specify the Jackknife method, the type would be `JK1`, `JKn`, or `JK2`. Additionally, the overall multiplier for JK1 is specified with the scale argument whereas the replicate specific multiplier ($\alpha_r) is specified with the rscales argument.
+
+Consider a case for the JK1 method where the multiplier, $(R-1)/R=19/20=0.95$ and the dataset had WT0 for the main weight and had 20 JK1 weights indicated WT1, WT2, ..., WT20, then the syntax would be
+
+```{r, eval=FALSE}
```

Comment:
You can remove the `eval=FALSE` here because you have it in the quarto comment style below.